### PR TITLE
Change travis.ci to github actions

### PR DESCRIPTION
**Pull request by T4rikA**
Co-authored-by: @arminWel

closes: #762 

Add tests.yml and setting_test.py for testing.
Updates requirements

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 -r requirements-test.txt
 django-debug-toolbar >=2.0, <2.1
-psycopg2==2.8.6
```
Comment (Paula-Kli) [inline requirements-dev.txt:3]: why is this removed in requirements dev?

Comment (arminWel) [reply requirements-dev.txt:3]: It is moved to  requirements-test.txt.

Comment (Paula-Kli) [reply requirements-dev.txt:3]: I see that but don't we need a Postgresql driver for development as well?

Comment (arminWel) [reply requirements-dev.txt:3]: Yep. But due to line 1, this is ensured.

**Review by Paula-Kli — APPROVED** [review]
